app/views/pages/shared/_searchbar.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_with url: "/search", method: :get do |form|  %>
+  <%= form.text_field :query %>
+  <%= form.submit "Search" %>
+<% end %>
```

Comment:
You don't need to specify `method: :get`, it's there by default
Also the url should be /arts because it's the arts controller that will deal with the search.

the text_field should be what is called :search instead of :query